moler/runner.py

```diff
@@ -211,16 +216,18 @@ def wait_for(self, connection_observer, connection_observer_future, timeout=None
         connection_observer_future.cancel()
         connection_observer.cancel()  # TODO: should call connection_observer_future.cancel() via runner
         connection_observer.on_timeout()
-        connection_observer.logger.log(logging.INFO,
```

Comment:
above - should not be return - function always returns None

Reply:
Test test_can_await_connection_observer_to_complete in test_thread_runner.py was modified after this change.